src/_projects/opensource-shows.md

```diff
@@ -5,4 +5,27 @@ lang: pt-BR
 state: current
 ---
 
-Aulas e workshops sobre algum tópico livre: ferramenta, ideologia, tecnologia, etc.
+O Open Source Show é um projeto que visa abrir um espaço para que todes -- incluindo não membres do grupo -- possam realizar
+apresentações sobre algum tema que seja relacionado a Cultura Livre!
+
+Podem ser feitos apresentações demonstrando alguma ferramenta FLOSS que você utilize no dia a dia, por exemplo, ou alguma
+apresentação mais teórica falando sobre os princípios da Cultura Livre, o *Copy Left*, etc...
+
+Basta ter criatividade!
+
+### Alguns dos Open Source Shows Passados
+
+<div style="display: block; text-align: center;">
+    <iframe width="480" height="315" src="https://www.youtube-nocookie.com/embed/EQJansnxksY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    <iframe width="480" height="315" src="https://www.youtube-nocookie.com/embed/GQykipTwaSI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    <iframe width="480" height="315" src="https://www.youtube-nocookie.com/embed/PImQb4FPrSg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    <iframe width="480" height="315" src="https://www.youtube-nocookie.com/embed/17vS1arTyPI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+</div>
+
+### Inscrição
+
+Para fazer uma apresentação no Open Source Show, basta escolher um tema e relacioná-lo a Cultura Livre e, depois, preencher [este formulário](https://cloud.gelos.club/apps/forms/s/4Ag2g5aenMS8TRqsQC7JAxC9).
+
+Seu tema será analisado pelo grupo e, estando aprovado, entraremos em contato para confirmar sua apresentação!
+
+Caso não seja aprovado, o grupo se compromete a retornar um *feedback* contendo o motivo da decisão.
```

Comment:
Será que a gente conseguiria deixar essa escolha de palavras com uma pegada um pouco menos "the council will decide your fate" e mais algo tipo "vamos conversar e melhorar juntos seu tema"? Algo tipo "peer-review"?

Acho que casos que a gente vai precisar _rejeitar_ mesmo um tema (e não só sugerir mudanças pra deixar ok) vão ser coisas bem raras, tipo troll mesmo. Então acho que podemos deixar o teor aqui um pouco mais "positivo"? Pra não fazer a pessoa pensar que existe altas possibilidades de ser rejeitado.

Acho que seria massa tentar sugerir, caso o tema não seja troll, que pessoas que fazem parte (ou querem fazer) possam receber feedback no próprio telegram; isso é legal pois podemos utilizar da expertise dos membros não-coordenadores do grupo pra trabalhar na apresentação.

Claro, se o assunto não tiver relevância alguma e não tiver como salvar, a gente recusa diretamente com a pessoa e nem leva isso pro telegram.

Uma sugestão, sinta-se livre pra mudar:

```suggestion
O grupo irá revisar seu tema, para verificar que se encaixa com a proposta do Open Source Show.

Se você já faz parte GELOS (ou queira fazer), convidamos que participe do grupo de trabalho no [telegram](https://t.me/gelos_geral) para uma discussão e revisão mais dinâmica. Caso contrário, daremos esse *feedback* por email.

Estando tudo certo, entraremos em contato para marcar a data e local da apresentação!
```

Reply:
Coloquei por email pq tô assumindo que vai ser essa a forma de contato com pessoas externas ao grupo que não querem fazer parte (por n quererem entrar no telegram ou coisa assim), sinta-se livre pra trocar pra algo mais genérico, se for melhor.